GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/ENERGIA.cfg

```diff
@@ -0,0 +1,1722 @@
+			@key = 0.87573 1
+			@key = 0.85892 1
+			@key = 0.84777 1
+			@key = 0.83665 1
+			@key = 0.82552 1
+			@key = 0.81439 1
+			@key = 0.80323 1
+			@key = 0.79207 1
+			@key = 0.78088 1
+			@key = 0.76976 1
+			@key = 0.75851 1
+			@key = 0.75051 1
+			@key = 0.74951 1
+			@key = 0.74851 1
+			@key = 0.74744 1
+			@key = 0.73665 1
+			@key = 0.72617 1
+			@key = 0.71568 1
+			@key = 0.70544 1
+			@key = 0.69526 1
+			@key = 0.68518 1
+			@key = 0.67521 1
+			@key = 0.66537 0.85
+			@key = 0.65101 0.70
+			@key = 0.64599 0.70
+			@key = 0.63649 0.70
+			@key = 0.62711 0.70
+			@key = 0.61782 0.70
+			@key = 0.60862 0.70
+			@key = 0.59952 0.70
+			@key = 0.59047 0.70
+			@key = 0.58152 0.70
+			@key = 0.57269 0.70
+			@key = 0.56392 0.70
+			@key = 0.55527 0.70
+			@key = 0.54672 0.70
+			@key = 0.53823 0.70
+			@key = 0.52976 0.70
+			@key = 0.52136 0.70
+			@key = 0.51302 0.70
+			@key = 0.50481 0.70
+			@key = 0.49671 0.70
+			@key = 0.48868 0.70
+			@key = 0.48077 0.70
+			@key = 0.47273 0.72
+			@key = 0.46476 0.85
+			@key = 0.45175 1
+			@key = 0.44869 1
+			@key = 0.44056 1
+			@key = 0.43241 1
+			@key = 0.42422 1
+			@key = 0.41690 1
+			@key = 0.40772 1
+			@key = 0.39941 1
+			@key = 0.39107 1
+			@key = 0.38278 1
+			@key = 0.37436 1
+			@key = 0.36586 1
+			@key = 0.35744 1
+			@key = 0.34891 1
+			@key = 0.34035 1
+			@key = 0.33176 1
+			@key = 0.32315 1
+			@key = 0.31453 1
+			@key = 0.30588 1
+			@key = 0.29723 1
+			@key = 0.28855 1
+			@key = 0.27987 1
+			@key = 0.27127 1
+			@key = 0.26252 1
+			@key = 0.25387 1
+			@key = 0.24531 0.992
+			@key = 0.23682 0.985
+			@key = 0.22841 0.972
+			@key = 0.2201 0.965
+			@key = 0.21185 0.961
+			@key = 0.20376 0.953
+			@key = 0.19575 0.947
+			@key = 0.18781 0.936
+			@key = 0.18005 0.928
+			@key = 0.17241 0.923
+			@key = 0.16493 0.910
+			@key = 0.1575 0.899
+			@key = 0.15011 0.890
+			@key = 0.14278 0.881
+			@key = 0.13548 0.875
+			@key = 0.12824 0.861
+			@key = 0.12115 0.853
+			@key = 0.11416 0.841
+			@key = 0.10726 0.833
+			@key = 0.10045 0.821			
+			@key = 0.0937 0.813
+			@key = 0.08704 0.791
+			@key = 0.08051 0.788
+			@key = 0.07206 0.781
+			@key = 0.06771 0.770
+			@key = 0.06155 0.754
+			@key = 0.05556 0.742
+			@key = 0.0497 0.737
+			@key = 0.04387 0.731
+			@key = 0.03804 0.621
+			@key = 0.03231 0.533
+			@key = 0.0269 0.490
+			@key = 0.02193 0.444
+			@key = 0.01751 0.395
+			@key = 0.0138 0.332
+			@key = 0.01085 0.264
+			@key = 0.00863 0.198
+			@key = 0.00682 0.162
+			@key = 0.00528 0.138
+			@key = 0.00396 0.118
+			@key = 0.00294 0.091
+			@key = 0.0022 0.066
+			@key = 0.00164 0.05
+			@key = 0.00126 0.033
+			@key = 0.00098 0.025
+			@key = 0.00073 0.022
+			@key = 0.00051 0.014
+			@key = 0.00041 0.012
+			@key = 0.00031 0.010
+			@key = 0.00021 0.009
+			@key = 0.00017 0.008
+			@key = 0.00014 0.007
+			@key = 0.00012 0.006
+			@key = 0.00009 0.005
+			@key = 0.00006 0.004
+			@key = 0.00001 0.003
+			@key = 0.000007 0.002
+		}
+    }

```

Comment:
If you set `engineType` and this engine has a config in RO, none of this is required.
(All of this sort of thing would go in the RO engine configs files, so that various mods could potentially supply this kind of engine and we don't have to duplicate all of the configs everywhere.)